cmd/performance-profile-creator/README.md

```diff
@@ -44,13 +44,15 @@ Depending on how must-gather directory was setup run the Performance profile Cre
 
 1. Option 1: Using must-gather output dir (obtained after running must gather manually)
    ```bash
-   podman run --entrypoint performance-profile-creator -v /path/to/must-gather-output:/must-gather:z\
-   quay.io/openshift-kni/performance-addon-operator:4.9-snapshot --must-gather-dir-path /must-gather > performance-profile.yaml
+   podman run --entrypoint performance-profile-creator -v /path/to/must-gather-output:/must-gather:z \
+   quay.io/openshift-kni/performance-addon-operator:4.9-snapshot --must-gather-dir-path /must-gather \
+   --reserved-cpu-count 20 --mcp-name worker-cnf --rt-kernel false > performance-profile.yaml
```

Comment:
I think we should use placeholders. Like:
```
--reserved-cpu-count $COUNT --mcp-name $NAME --rt-kernel $FLAG
```

Reply:
I thought so too, however in the following example "Running Performance Profile Creator using Wrapper script" exact parameters were used, so I just recycled those ones.
```
./hack/run-perf-profile-creator.sh -t must-gather.tar.gz -- --mcp-name=worker-cnf --reserved-cpu-count=20 \
--rt-kernel=false --split-reserved-cpus-across-numa=true --topology-manager-policy=restricted \
--power-consumption-mode=low-latency > performace-profile.yaml
```

If you want I change both, or just the I'm already fixing, or leave it as it.

Reply:
Let's hear what @swatisehgal prefers. I don't have strong opinions either way, but I don't think that setting values without clearly signaling they are examples is a clear benefit